src/bin/ion/commands/inspect.rs

```diff
@@ -223,6 +223,8 @@ struct IonInspector<'a, 'b> {
     skip_complete: bool,
     limit_bytes: usize,
     hide_expansion: bool,
+    ephemeral_depth: usize,
```

Comment:
Ephemeral values are now grayed out in the output. This tracks the level of traversal depth where the ephemeral value began so we know when to restore color.